packages/sst/src/constructs/SsrSite.ts

```diff
@@ -513,6 +553,47 @@ export abstract class SsrSite extends Construct implements SSTConstruct {
     };
   }
 
+  /** @internal */
+  protected get ssrOrigin() {
+    if (this._singletonSsrOrigin) return this._singletonSsrOrigin;
+
+    // TODO
+    console.log({ support: this.supportsStreaming() });
+    const { timeout } = this.props;   
+    const fnUrl = this.serverLambdaForRegional!.addFunctionUrl({
+      authType: FunctionUrlAuthType.NONE,
+      invokeMode: this.supportsStreaming()
+        ? InvokeMode.RESPONSE_STREAM
+        : undefined,
+    });
+    
+    
+    this._singletonSsrOrigin = new HttpOrigin(Fn.parseDomainName(fnUrl.url), {
+      readTimeout:
+        typeof timeout === "string"
+          ? toCdkDuration(timeout)
+          : CdkDuration.seconds(timeout),
+    });
+    return this._singletonSsrOrigin;
+  }
+
+  /** @internal */
+  protected get cachePolicy() {
+    if (this._singletonCachePolicy) return this._singletonCachePolicy;
+    this._singletonCachePolicy = this.buildServerCachePolicy();
```

Comment:
The logic for `buildServerCachePolicy` should probably be moved into this getter if this methodology is acceptable.